app/templates/dashboard.html

```diff
@@ -69,13 +71,8 @@ <h1 class="govuk-heading-xl">Your applications</h1>
     'rows': ns.rows
     }) }}
 {% endif %}
-<form method="POST" action={{ url_for('routes.new')}}>
-    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
-    <input type="hidden" name="account_id" value="{{ account_id }}">
-    <input type="hidden" name="fund_id" value="{{ fund_id }}">
-    <input type="hidden" name="round_id" value="{{ round_id }}">
-    {{ govukButton({
-    'text': "Start new application"
-    }) }}
-</form>
+
+{{startApplicationButton(form, url_for('routes.new'), applications=applications)}}
+
 {% endblock %}
+ 
```

Comment:
Is this a spare line or space or something?

Reply:
I ran pre commit hook on the whole service by mistake 😆